src/promptflow/promptflow/_cli/_pf/_service.py

```diff
@@ -45,52 +52,100 @@ def get_app(environ, start_response):
     return app.wsgi_app(environ, start_response)
 
 
-def add_start_service_action(subparsers):
-    """Add action to start pfs."""
-    start_pfs_parser = subparsers.add_parser(
-        "start",
-        description="Start promptflow service.",
-        help="pfs start",
+def add_service_parser(subparsers):
+    """Add service parser to the pf subparsers."""
+    service_parser = subparsers.add_parser(
+        "service",
+        description="Manage promptflow service.",
```

Comment:
Introduce a bit on what pf service does:  provide chat and trace ui.

Reply:
updated